File: Ex10.java
package Ex10;

class Calculadora {

    public double somar(double a, double b) {
        return a + b;
    }

    public double somar(double a, double b, double c) {
        return a + b + c;
    }

    public static void main(String[] args) {
        Calculadora calculadora = new Calculadora();
        
        double soma2 = calculadora.somar(5.0, 3.0);
        double soma3 = calculadora.somar(5.0, 3.0, 2.0);
        
        System.out.println("Soma de 5.0 e 3.0: " + soma2);
        System.out.println("Soma de 5.0, 3.0 e 2.0: " + soma3);
    }
}

File: ex10.go
package main

import (
	"fmt"
)

type Calculadora struct{}

func (c Calculadora) SomarDoisNumeros(a float64, b float64) float64 {
	return a + b
}

func (c Calculadora) SomarTresNumeros(a float64, b float64, c float64) float64 {
	return a + b + c
}

func main() {
	calculadora := Calculadora{}

	resultado1 := calculadora.SomarDoisNumeros(5, 10)
	fmt.Printf("Soma de dois números: %.2f\n", resultado1)

	resultado2 := calculadora.SomarTresNumeros(5, 10, 15)
	fmt.Printf("Soma de três números: %.2f\n", resultado2)
}
